sound/soc/sof/intel/pci-icl.c

```diff
@@ -61,7 +64,10 @@ static const struct sof_dev_desc jsl_desc = {
 		[SOF_IPC] = "intel/sof-tplg",
 		[SOF_INTEL_IPC4] = "intel/avs-tplg",
 	},
-	.default_fw_filename = "sof-jsl.ri",
+	.default_fw_filename = {
+		[SOF_IPC] = "sof-icl.ri",
+		[SOF_INTEL_IPC4] = "dsp_basefw.bin",
```

Comment:
all APL + will share the same name "dsp_basefw.bin" ?

Reply:
o, I got it, we will have different directory

Reply:
yep